code/__DEFINES/dcs/signals/signals_janitor.dm

```diff
@@ -15,3 +15,6 @@
 #define COMSIG_TURF_IS_WET "check_turf_wet"
 ///(max_strength, immediate, duration_decrease = INFINITY): Returns bool.
 #define COMSIG_TURF_MAKE_DRY "make_turf_try"
+
+/// Sent to an object when it has been cleaned by /datum/component/cleaner
+#define COMSIG_OBJECT_CLEANED "object_cleaned"
```

Comment:
I regret to inform you this already exists, it's `COMSIG_COMPONENT_CLEAN_ACT`

Reply:
Works, but I can't for the life of me find where it's sent from. Thanks though